astropy/visualization/wcsaxes/tests/test_images.py

```diff
@@ -1398,3 +1400,27 @@ def fruit_format(value, **kwargs):
     ax.coords[0].set_major_formatter(double_format)
     ax.coords[1].set_major_formatter(fruit_format)
     return fig
+
+@figure_test(tolerance=1)
```

Comment:
Please see https://docs.astropy.org/en/latest/development/testguide.html#image-tests-with-pytest-mpl for how to set up the "truth" for this new image test. Thanks!